contracts-core/periphery-artifacts/UmbraBatchSend.sol/UmbraBatchSend.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
+  "_format": "hh-sol-dbg-1",
```

Comment:
What are all these new files in `contracts-core`? I'd think we don't need to modify this folder, especially since it seems there's BatchSend artifacts here?

Reply:
I was able to use `/out` as the artifact but I think I'm having `ethers` version problem with the typechain file. Because if I change
```
import { UmbraBatchSend as BatchSendContract } from '@umbra/contracts-core/periphery-typechain/UmbraBatchSend';
```
to 
```
...from '@umbra/contracts-periphery/periphery-typechain/UmbraBatchSend';
```
I'm getting type errors. Also getting type errors when autogenerating the files using `yarn run typechain --target ethers-v5 --out-dir ./typechain './out/UmbraBatchSend.sol/UmbraBatchSend.json'` and when using
```
...from '@umbra/contracts-periphery/typechain/UmbraBatchSend';
```
Tried to play around with the `contract-periphery` folder's `package.json` by installing the `^5.6.8` version of `ethers` and other few older versions of `typechain` and `typechain/ethers-v5` to no avail

Reply:
What if we just blow away all the typechain stuff for now since it's more of a nice to have, get things working, and we can improve the typings later? I'm getting a bit confused by all the autogenerated files, and I think @wildmolasses may have setup some lerna build order config for ensuring typechain gets generated in `contracts-core` before the umbra-js package is built, so might need to ask him for help here too

Reply:
Oh I didn't know they were nice to haves. I was looking at the implementation for Umbra's `send` and thought they were essential.

Reply:
To expand on why typechain is just a nice to have, and not essential: By default if you create a new contract in ethers with `myContract = new Contract(...)` you get an object of the `Contract` type, i.e. an instance of the `Contract` class. So let's say you pass in an ERC-20 ABI. Then you can call `myContract.name()` as expected. But one downside here is that you have to tell typescript that you expect a `string` to be returned, it doesn't know what that method is supposed to return.

You may accidentally call `myContract.nameeee()`, which of course doesn't exist in an ERC-20. Your code will compile, then you'll get a runtime error since that method doesn't exist on the called contract.

Using Typechain solves these problems by giving you classes such as `UmbraBatchSend` which extend the default ethers `Contract` class based on the ABI. So now (1) your TS code knows `name` returns a string, and (2) your TS code knows all available methods and won't compile if you try calling a method that doesn't exist.

Probably some more info at https://github.com/dethcrypto/TypeChain

Reply:
Awesome, I'm starting to become a fan of type safety. But removed them for now!

Reply:
Type safety is 1000% better than no type safety 😛 but yea if it's causing problems or being weird let's get the core part working first and we can always follow up in a separate PR. Leaving this unresolved until we either decide to implement it in this PR at the end or pull out into a separate issue

Reply:
Created a separate issue for it! https://github.com/ScopeLift/umbra-protocol/issues/457